openedx/adg/lms/user_authn_override/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+`user_authn_override` app overrides the existing authentication forms by creating and returning new forms with the
```

Comment:
Basically the app is written to override any kind of implementation of user_authn core app and as an example we can mention the we have overrode forms